packages/flutter/lib/src/gestures/scale.dart

```diff
@@ -226,7 +225,28 @@ class ScaleGestureRecognizer extends OneSequenceGestureRecognizer {
   ScaleGestureRecognizer({
     Object? debugOwner,
     PointerDeviceKind? kind,
-  }) : super(debugOwner: debugOwner, kind: kind);
+    this.dragStartBehavior = DragStartBehavior.down,
```

Comment:
I defaulted this to `down`, which is the current behavior, even though DragGestureDetector defaults to `start`.

Reply:
Doesn't seem to break any local test when I change the default to start. In this case should we keep it consistent?